### PR TITLE
Fix scrolling edges

### PR DESCRIPTION
**Pull request by yukiej**
Also:

- fixed View text to not overlap with long fieldnames
- date picker aligned to impact column
- cal automatically opens with CUSTOM selected in date picker

### CODE REVIEW

**Comment by yukiej** [comment]
Build: https://builds.cask.co/browse/CDAP-URUT202-1
RAT check: https://builds.cask.co/browse/CDAP-DRC9654

File: cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx [hunk 1]
```diff
@@ -85,7 +90,7 @@ function FllField({ field, isActive, classes }: IFieldProps) {
       data-hovering={isHovering}
       data-target={isTarget}
     >
-      {field.name}
+      <span className={classes.fieldname}>{field.name}</span>
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx:93]: did this work? span is an inline element, so it doesn't have a width. They shouldn't have an overflow

File: cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx [hunk 2]
```diff
@@ -50,16 +50,19 @@ const styles = (theme) => {
...
       ' & .grid-row:hover': {
         backgroundColor: theme.palette.grey[700],
-        cursor: 'pointer',
       },
       ' & .grid-row.selected': {
         backgroundColor: theme.palette.yellow[200],
         color: theme.palette.orange[50],
         fontWeight: 'bold',
         cursor: 'unset',
+        gridTemplateColumns: '1fr auto !important',
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx]: is the `!important` necessary? I think seems like it's already set by grid row.

File: cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx (hunk 1 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx:93]: It did work, but I don't know why...

File: cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx (hunk 2 above)
Comment (yukiej) [reply cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx]: If the !important isn't there, the columns get set by grid-row which doesn't quite work. Will add a comment along those lines.

Comment (yukiej) [reply cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/index.tsx]: That said, this specific line is redundant, so removing

File: cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx:93]: oh grid-row is setting display grid.. so all the children will not be inline anymore

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by yukiej** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-URUT202-4

**Pull request merged**